GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3022,15 +3024,7 @@ private void RevisionInfo_CommandClick(object sender, CommitInfo.CommandEventArg
 
         private void SubmoduleToolStripButtonClick(object sender, EventArgs e)
         {
-            var button = sender as ToolStripMenuItem;
-
-            if (button == null)
-                return;
-
-            if (button.Tag is GitModule)
-                SetGitModule(this, new GitModuleEventArgs(button.Tag as GitModule));
-            else
-                SetWorkingDir(button.Tag as string);
+            SetWorkingDir((sender as ToolStripMenuItem).Tag as string);
```

Comment:
Will this NRE if the sender isn't a ToolStripMenuItem (it would silently return before). In other words, are we SURE that sender is always one?